portal/static/js/src/profile.js

```diff
@@ -2771,12 +2768,38 @@ export default (function() {
                     $(".put-roles-error").html("A role must be selected.");
                     return false;
                 }
+                var isChecked = $(event.target).is(":checked");
+                var checkedRole = event.target.value;
```

Comment:
Perhaps a better name for this is 'changedRole', since the event could be the user either checking it, or unchecking it? That would make this easier to read...